Dockerfile.centos7

```diff
@@ -3,6 +3,7 @@ FROM registry.centos.org/centos/centos:7
 RUN yum update -y && PKGS="centos-release-ansible-28 centos-release-scl-rh" && \
     yum install -y $PKGS && rpm -V $PKGS && \
     PKGS="ansible epel-release python2-jmespath rsync" && yum install -y $PKGS && rpm -V $PKGS && \
+    PKGS="python-netaddr" && yum install -y $PKGS && rpm -V $PKGS && \
```

Comment:
you can just add this to the list above e.g. `PKGS="ansible epel-release python2-jmespath rsync python-netaddr" && yum install -y $PKGS && rpm -V $PKGS` - will be faster

Reply:
done